demos/README.md

```diff
@@ -169,6 +169,10 @@ cd build
 ```sh
 make
 ```
+Or you can use `cmake --build` tool:
+```sh
+cmake --build .
+```
```

Comment:
Do we need to provide both alternatives? I think we can just promote the `cmake --build` approach. It has no downsides (compared to running `make` directly) that I can think of, and it works with non-make generators as well.

Reply:
Ok, removed `make` approach.